packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/RichTextFieldDisplay.tsx

```diff
@@ -0,0 +1,11 @@
+import { useTextFieldDisplay } from '@/object-record/record-field/meta-types/hooks/useTextFieldDisplay';
+import { getFirstNonEmptyLineOfRichText } from '@/ui/input/editor/utils/getFirstNonEmptyLineOfRichText';
+import { PartialBlock } from '@blocknote/core';
+
+export const RichTextFieldDisplay = () => {
+  const { fieldValue } = useTextFieldDisplay();
+  const parsedField =
+    fieldValue === '' ? null : (JSON.parse(fieldValue) as PartialBlock[]);
```

Comment:
is this needed? I feel '' should not be a valid rich text value. Why do we consider it?

Reply:
Yes I thought you'd want it to be non nullable like strings

TODO: add backfill command to convert empty string to NULL